scaladoc-testcases/src/tests/functionTypeSignatures.scala

```diff
@@ -0,0 +1,10 @@
+package tests.functionTypeSignatures
+
+type A = ((Int, Int)) => Int
+
+type B = (Int | String) => Int
```

Comment:
`|` and `&` type operators bind stronger than `=>` so it's a judgment call it they should be in parentheses.

Reply:
I think for the sake of clarity we should keep them in parentheses

Reply:
Ok, makes sense